spec/index.bs

```diff
@@ -10359,6 +10388,36 @@ dictionary GPURenderPassColorAttachment {
     where |descriptor| is |view|.{{GPUTextureView/[[descriptor]]}}.
 </div>
 
+<div algorithm>
+    <dfn abstract-op>Calculating total color attachment depth</dfn>(|formats|)
+
+    **Arguments:**
+
+    - sequence<{{GPUTextureFormat}}> |formats|
+
+    **Returns:** {{GPUSize32}}
+
+    1. Let |total| be 0.
+    1. For each |format| in |formats|
+        1. [=Assert=]: |format| is a [=color renderable format=].
+        1. Let |texelBlockSize| be the [=Texel block size=] of |format|.
+        1. Let |componentSize| be the [=component width=] of |format|.
+        1. If |componentSize| is [=mixed component width=]:
+            1. Set |texelBlockSize| to 8.
+            1. Set |componentSize| to 4.
+        1. Otherwise, divide |componentSize| by 8 (to convert bits to bytes).
+        1. If |format| is any of the following formats, then set |texelBlockSize| to 8.
+            - {{GPUTextureFormat/"rgba8unorm-srgb"}}
+            - {{GPUTextureFormat/"rgba8unorm"}}
+            - {{GPUTextureFormat/"bgra8unorm-srgb"}}
+            - {{GPUTextureFormat/"bgra8unorm"}}
+            - {{GPUTextureFormat/"rgba8snorm"}}
```

Comment:
optional: It would be more straightforward if these component size and texel size special cases were part of the texture format table, e.g. "attachment byte cost" and "attachment byte alignment" - though it would also make the table a bit bigger. I'm OK with either.

Reply:
https://github.com/gpuweb/gpuweb/pull/3369 would make the wider table less of a problem.